UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -73,6 +73,8 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double UsSurveyFeetPerHourInOneMeterPerSecond => 1.1811E4;
 
+        protected override double MachInOneMeterPerSecond => (1.0 / 340.29);
```

Comment:
This should be a calculated number, not a calculation.

The idea is to look up this value from a reliable source, such as another unit converter, wikipedia, wolfram, google and similar, in order to verify we got the conversion functions right - and in particular, got the _direction_ of the from/to conversion functions right.

Reply:
Thanks @angularsen.  I have pushed what I hope are the requested changes to tjrtfarnham:add-mach-speed-unit.  Do I need to create another pull request.  Sorry, I am a total newbie when it comes to using github.

Reply:
Perfect, that is the way to go. The new commits arrive in the pull request for me to review 👍